habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/InstallDependenciesMojo.java

```diff
@@ -386,6 +388,16 @@ protected void performPendingDependencyReplacements(Map<String, TomlReplacementT
         }
     }
 
+    /**
+     * Handles escaping with double quotes only if the value is not an array.
+     *
+     * @param valueToEscape value to potentially escape
+     * @return value ready to write to toml file
+     */
+    protected static String escapeTomlRightHandSide(String valueToEscape) {
+        return (!valueToEscape.contains("{")) ? DOUBLE_QUOTE + valueToEscape + DOUBLE_QUOTE : valueToEscape;
```

Comment:
Q: For clarity, this isn't actually checking if there is an array but a table instead? 

https://toml.io/en/v1.0.0#inline-table